tests/test_set_keyspace_initial_tablets.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from typing import List, Optional
+
+from ccmlib.node import Node
+
+
+@pytest.mark.parametrize("stress_options, initial_tablets, datacenter, rf, expected", [
+    # typical case
+    (['write', 'n=0', 'no-warmup', '-schema', 'replication(factor=1)', '-rate', 'threads=1'], 8, None, 3,
+     ['write', 'n=0', 'no-warmup', '-schema', 'replication(strategy=NetworkTopologyStrategy,datacenter1=1,initial_tablets=8)', '-rate', 'threads=1']),
```

Comment:
looking at those example, I don't think we should be changing the `strategy`, that might be changing the test logic more then we intend.

SimpleStrategy should accept `initial_tablets`, and if not, I think it's a bug

Reply:
`initial_tablets` are not planned for `SimpleStrategy`, at least not in near future. 

I don't see any other option than change this to `NetworkTopologyStrategy`.
Shall we split this change to first adapt dtests to NTS?